pages/templates/ebraj/index.tsx

```diff
@@ -161,7 +161,7 @@ export const HeroSection = () => {
         <div className="h-60 bg-gradient-to-br from-violet-500 to-violet-600 blur-[200px] dark:from-sky-900 dark:to-sky-950"></div>
         <div className="h-60 bg-gradient-to-r from-violet-500 to-violet-600 blur-[200px] dark:from-sky-800 dark:to-sky-900"></div>
       </div>
-      <main className="text-center">
+      <main className="text-center flex flex-col">
```

Comment:
If you had added `justify-center items-center` here , it would have achieved the same result without the need to make changes to the button. Nevertheless, I believe the output remains consistent in both scenarios👍.

Reply:
@tanpreetjolly Can you please do this changes instead of current change you have done.